### DISCUSSION TITLE
Can I submit the project without TeamsFx?

### DISCUSSION THREAD

**Discussion by YuriyMorozyuk95** [microsoft/hack-together-teams#42] [Q&A] [answered]
Unfortinatly security in my office block preparing Teams App Dependecies(

It possible submit the project where I manullay create Asp.Net Core project use Microsoft.Bot.Builder.Integration.AspNet.Core 4.20, and Azure.Open.AI SDK without TeamsFx.
And manully configure all resources in Azure?

It would like to use Bot Framworkj Emulator for debug, and later publish app to teams old way.

Best regards, Yurii

**Comment by waldekmastykarz** [comment, marked as answer]
Absolutely! As long as your app is an app for Microsoft Teams, we don't have specific requirements as to which dependencies you have to use.

That said, I'd love to learn more about your constraints. Could you please share some more information with us about what's blocked and preventing you from using TeamsFx?

/cc: @therealjohn

**Reply by YuriyMorozyuk95** [reply]
Yes sure, this issue related to this discussion: https://github.com/microsoft/hack-together-teams/discussions/10

Bescially according I am not able to connect to my Office 365 accaunt from Visual Studio.
When I am building TeamsToolkit 5 solution, FxTeams add pre debug task step, for validation do accaunt is connected. I tryed someshow disable this task, and run app without opening teams. But as I understood this logic is part FxTeams, and not possible to disble it.

For this reasons I make decission to not use FxTeams and develope app old way.

**Reply by therealjohn** [reply]
For development, we recommend using a Microsoft 365 Developer account. This lets you have your own free subscription and sandbox environment to create apps with. Are you able to use that instead of your business account?

However, there is no requirement to use Teams Toolkit for the hackathon.

**Reply by YuriyMorozyuk95** [reply]
I use Microsoft 365 Developer account, but unfortinatly I am not able to connect it with Visual Studio, and I do manually all configuration and deployment in azure.
